distributed/tensor_parallelism/two_d_parallel_example.py

```diff
@@ -1,25 +1,23 @@
-import argparse
 
```

Comment:
nit: file name, I think let's call it fsdp_tp_example.py, `two_d_parallel_example` is too vague

Reply:
makes sense - updated file name and references (i.e. updated default for run_example.sh)